src/TimeService.Mappers/Requests/Interfaces/IEditWorkTimeMapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.TimeService.Mappers.Interfaces;
+using LT.DigitalOffice.TimeService.Models.Db;
+using LT.DigitalOffice.TimeService.Models.Dto;
+
+namespace LT.DigitalOffice.TimeService.Mappers.Requests.Interfaces
+{
+    [AutoInject]
+    public interface IEditWorkTimeMapper : IMapper<EditWorkTimeRequest, DbWorkTime>
```

Comment:
As I know, we considered to name mapper after entity it returns. Can you, please, rename them all?

Reply:
Renamed